src/generic_ui/polymer/cloud-install.ts

```diff
@@ -83,6 +83,7 @@ Polymer({
     }).then(() => {
       this.closeOverlays();
       this.$.successOverlay.open();
+      ui.model.globalSettings.shouldHijackDO = false;
```

Comment:
@trevj I think I may have missed a `ui_context.core.updateGlobalSettings(ui_context.model.globalSettings)` call here, yeah? Will fix in #2500.